ScoutSuite/providers/azure/rules/findings/network-security-groups-rule-inbound-service-mssql.json

```diff
@@ -0,0 +1,34 @@
+{
+    "description": "Security Group Rules Allowing Inbound MsSQL Access",
+    "rationale": "MsSQL (1433) inbound access should not be allowed to a network security group.",
+    "compliance": [
+        {
+            "name": "CIS Microsoft Azure Foundations",
+            "version": "1.0.0",
```

Comment:
No v1.2?

Reply:
No it doesn't exist in v1.2.0, so I wasn't sure if I should keep it or not. I seperated it from the other 2 RDP and SSH for the update